plugins/rebalance/transaction.go

```diff
@@ -0,0 +1,58 @@
+package rebalance
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+)
+
+type actionType uint16
+
+const (
+	actionStart actionType = iota
+	actionStop
+)
+
+func rebalanceCommand(c transaction.TxnCtx, cmd actionType) error {
+	var rinfo rebalanceapi.RebalInfo
+	err := c.Get("rinfo", &rinfo)
+	if err != nil {
+		return err
+	}
+
+	rebalanceProcess, err := NewRebalanceProcess(rinfo)
+	if err != nil {
+		return err
+	}
+
+	switch cmd {
+	case actionStart:
+		var r rebalanceapi.RebalInfo
+		// Reset all variables
+		// TODO: glusterdVolResetStats resets few variables to zero
+		// These variables updates in dht code and then need to be updated in Rebalance info struct
+		glusterdVolInfoResetStats(&r)
+		err = daemon.Start(rebalanceProcess, true)
+
+		// TODO: Need to create glusterd rebalance rpc connection
+		// Create rpc client connection
+		/*_, err := daemon.GetRPCClient(rebalanceProcess)
+		if err != nil {
+			c.Logger().WithError(err).WithField(
+				"volume", rinfo.Volname).Error("rpc creation failed")
+			return err
+		}*/
+
+	case actionStop:
+		err = daemon.Stop(rebalanceProcess, true)
```

Comment:
Is it just stopping the Rebalance daemon? I think if rebalance will complete ongoing transfers and then Stop.

@nbalacha please confirm.